src/cli/cli_srp_server.cpp

```diff
@@ -178,6 +245,26 @@ template <> otError SrpServer::Process<Cmd("ttl")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli srp server lease (get,set)
+ * @code
+ * srp server lease 1800 7200 86400 1209600
+ * Done
+ * @endcode
+ * @code
+ * srp server lease
+ * min lease: 1800
+ * max lease: 7200
+ * min key-lease: 86400
+ * max key-lease: 1209600
+ * Done
+ * @endcode
+ * @cparam srp server lease [@ca{min-lease}] [@ca{max-lease}] [@ca{min-key-lease}] [@ca{max-key-lease}]
```

Comment:
Can we maybe change it to 
```suggestion
 * @cparam srp server lease [@ca{min-lease} @ca{max-lease} @ca{min-key-lease} @ca{max-key-lease}]
```
Basically we can either issue `srp server lease` to get the lease values, or if we want to set the values we then need to provide all four values.